src/app/utils/utils.js

```diff
@@ -223,6 +223,10 @@ const basicQuery = (props = {}) => {
     pageQuery = page && page !== '1' ? `&page=${page}` : pageQuery;
     pageQuery = page === '1' ? '' : pageQuery;
 
+    const completeQuery = `${searchKeywordsQuery}${filterQuery}${sortQuery}${fieldQuery}${pageQuery}`;
+
+    if (!completeQuery) return null;
+
     return `q=${searchKeywordsQuery}${filterQuery}${sortQuery}${fieldQuery}${pageQuery}`;
```

Comment:
You have a chance to reuse `completeQuery` here.

Reply:
Could even replace 228-30 with return completeQuery ? q=${completeQuery} : null